src/append_only_data.rs

```diff
@@ -969,4 +998,15 @@ mod tests {
         let decoded = unwrap!(self::Address::decode_from_zbase32(&encoded));
         assert_eq!(address, decoded);
     }
+
+    #[test]
+    fn append_unseq_test() {
+        let mut data = UnpubUnseqAppendOnlyData::new(XorName(rand::random()), 10);
+        let key1 = b"KEY1".to_vec();
+        let val1 = b"VALUE1".to_vec();
+        let key2 = b"KEY2".to_vec();
+        let val2 = b"VALUE2".to_vec();
+        let tup = [(key1, val1), (key2, val2)].to_vec();
+        assert_eq!(unwrap!(data.append(tup)), ());
```

Comment:
Can be
```suggestion
        assert!(data.append(tup).is_ok());
```

Reply:
I agree the `assert_eq` in unnecessary, but I generally prefer unwrapping results in tests rather than just checking `is_ok()` since if it fails, you see the error value.  Not a big deal though.